pypowsybl/network.py

```diff
@@ -4124,6 +4124,228 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def create_load_bay(self, busbar_section_id: str, load_position_order: int, df: _DataFrame,
+                        load_direction: str = "BOTTOM", throw_exception: bool = False,
+                        reporter: _Reporter = None, **kwargs: _ArrayLike) -> None:
+        """
+        Creates a load, connects it to the network and creates the topology. The voltage level containing the
+        busbar section should be described in node/breaker topology. The load is connected to the busbar with a
+        breaker and a closed disconnector. If the network has position extensions, the load will also be connected
+        to every parallel busbar section with an open disconnector.
+
+        Args:
+            busbar_section_id: id of the busbar section to which the injection will be connected with a closed
+            disconnector.
+            load_position_order: the order of the load, will fill the ConnectablePosition extension
```

Comment:
For now it is not possible to use CreateFeederBay without the order. But it will be done in a future version and then it will be possible to change it here too.

Reply:
Ok